index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute for Cristiano Ronaldo</title>
+    <link rel="stylesheet" href="tribute.css">
+</head>
+<body>
+    <div id="main">
+        <h1 id="title">CR7 - The greatest of all time(GOAT)</h1>
+        <div id="img-div">
+            <a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/Cristiano_Ronaldo">
+            <img id="image" src="https://cdn.resfu.com/media/img_news/imagen-de-presentacion-de-cristiano-ronaldo-en-case-esports--twitter---case-esports.png?size=1000x&lossy=1&ext=jpeg"
```

Comment:
I suggest you increase the margin on the left-hand side. because it doesn't look professional 
Also, I suggest that you consider the border-radius

Reply:
- [ ] Why do you suggest the border-radius to be considered? Please give reasons for requesting changes. This helps the student learn & improve their technical knowledge. 😄